backend/src/docs/v1.api-spec.yaml

```diff
@@ -139,14 +179,16 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  /kcConfigForm:
+  /keyCloak/configForm:
```

Comment:
If we're adopting `/webade/{webAdeEnv}`, we should also consider doing the same to the API shape for the POST configForm endpoint as well. Both the `*ConfigForm` actions are currently using a required ClientEnvironment body parameter for it so it may be worth embedding it into the path.

Reply:
See above